app/Http/Controllers/Api/v1/DefaultThumbnailApiController.php

```diff
@@ -49,22 +52,21 @@ public function upload(StoreDefaultThumbnail $request, Campaign $campaign)
      * @return \Illuminate\Http\JsonResponse
      * @throws \Illuminate\Auth\Access\AuthorizationException
      */
-    public function delete(DestroyDefaultThumbnail $request, Campaign $campaign)
-    {
-        if ($campaign->premium()) {
+    public function delete(DestroyDefaultThumbnail $request, Campaign $campaign) {
+        if ($campaign->premium()){
```

Comment:
Wrong check, and same issue: make the if `!` and return early :)

Reply:
You managed to review the changes before I fixed te merge conflicts haha